--- fix/aclocal.m4	2020-10-12 20:40:17.000000000 +0000
+++ fix/aclocal.m4	2024-08-03 22:14:08.464531457 +0000
@@ -139,10 +139,10 @@
 		AC_DEFINE([USE_TCL_STUBS], [1], [Define if you are using the Tcl Stubs Mechanism])
 
 		TCL_STUB_LIB_SPEC="`eval echo "${TCL_STUB_LIB_SPEC}"`"
-		LIBS="${LIBS} ${TCL_STUB_LIB_SPEC}"
+		#LIBS="${LIBS} ${TCL_STUB_LIB_SPEC}"
 	else
 		TCL_LIB_SPEC="`eval echo "${TCL_LIB_SPEC}"`"
-		LIBS="${LIBS} ${TCL_LIB_SPEC}"
+		#LIBS="${LIBS} ${TCL_LIB_SPEC}"
 	fi
 
 	TCL_INCLUDE_SPEC="`eval echo "${TCL_INCLUDE_SPEC}"`"
@@ -170,7 +170,7 @@
 	fi
 	AC_SUBST(TCL_PACKAGE_PATH)
 
-	AC_SUBST(LIBS)
+	#AC_SUBST(LIBS)
 ])
 dnl Usage:
 dnl    DC_TEST_SHOBJFLAGS(shobjflags, shobjldflags, action-if-not-found)
